### PR TITLE
Abort if label is not applied to pods

### PR DESCRIPTION
**Pull request by ramkrsna**
Signed-off-by: Ramakrishna Reddy Yekulla <rreddy@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/220)

### CODE REVIEW

**Comment by ramkrsna** [comment]
The above patch resolved #219

File: deploy/gk-deploy [hunk 1]
```diff
@@ -451,6 +451,10 @@ if [[ $GLUSTER -eq 1 ]] && [[ ${LOAD} -eq 0 ]]; then
   while read -r node; do
     debug "Marking '${node}' as a GlusterFS node."
     eval_output "${CLI} label nodes ${node} storagenode=glusterfs 2>&1"
+    if [[ ${?} -ne 0 ]]; then
+        output "Failed to Label the node"
+    abort
```
Comment (raghavendra-talur) [inline deploy/gk-deploy]: indentation

File: deploy/gk-deploy [hunk 2]
```diff
@@ -451,6 +451,10 @@ if [[ $GLUSTER -eq 1 ]] && [[ ${LOAD} -eq 0 ]]; then
   while read -r node; do
     debug "Marking '${node}' as a GlusterFS node."
     eval_output "${CLI} label nodes ${node} storagenode=glusterfs 2>&1"
+    if [[ ${?} -ne 0 ]]; then
+        output "Failed to Label the node"
```
Comment (raghavendra-talur) [inline deploy/gk-deploy]: Useful to print the ${node} here.

File: deploy/gk-deploy [hunk 3]
```diff
@@ -451,6 +451,10 @@ if [[ $GLUSTER -eq 1 ]] && [[ ${LOAD} -eq 0 ]]; then
   while read -r node; do
     debug "Marking '${node}' as a GlusterFS node."
     eval_output "${CLI} label nodes ${node} storagenode=glusterfs 2>&1"
+    if [[ ${?} -ne 0 ]]; then
+      output "Failed to Label the ${node}"
```
Comment (jarrpa) [inline deploy/gk-deploy]: Reword to `Failed to label node: ${node}`

Comment (obnoxxx) [inline deploy/gk-deploy]: ack

**Comment by obnoxxx** [comment]
superseded by #223

**Pull request closed without merging**